build-security/beats
# opabeat agent integration v1

## What does this PR do?

* Adds the ability for kubebeat to run as part of the fleet managed elastic agent
* Updated [Justfile ](https://github.com/build-security/beats/blob/master/kubebeat/JUSTFILE)with one line build/deploy commands for is of development
* Updated Readme with justfile commands. local environment setup & kind k8s local cluster installation

## Why is it important?



## Checklist




## Author's Checklist



## How to test this PR locally



Follow the Readme to build & deploy elastic-agent & setup the kibana dataview `logs-k8s_cis.result-*`.
Afterwards check that an index named `logs-k8s_cis.result-default` is created and events are viewable in discover under 
it's data view.

## Related issues


- 

## Use cases



- Run kubebeat via fleet managed agent 

## Screenshots



![image](https://user-images.githubusercontent.com/16253938/146836755-6d4c9385-2b18-4280-b906-8c29fa66ec23.png)

![image](https://user-images.githubusercontent.com/16253938/146836829-59b72dc3-401b-43dc-819f-68582681c2cf.png)

![image](https://user-images.githubusercontent.com/16253938/146836858-5173653c-c6bc-4fd0-ab5c-ea6db52dbed4.png)


## Logs
